pyrobosim/pyrobosim/navigation/__init__.py

```diff
@@ -9,4 +9,31 @@
 from .occupancy_grid import *
 
 # Planners
-from .path_planner import *
+from .a_star import AStarPlanner
+from .rrt import RRTPlanner
+from .prm import PRMPlanner
+from .world_graph import WorldGraphPlanner
+
+
+PATH_PLANNERS_MAP = {
+    "astar": AStarPlanner,
+    "rrt": RRTPlanner,
+    "prm": PRMPlanner,
+    "world_graph": WorldGraphPlanner,
+}
+
+
+def get_planner_class(planner_type):
+    """
+    Helper function that returns a path planner.
+
+    :param planner_type: The type of path planner.
+    :type planner_type: str
+    :return: The class corresponding to the planner type specified.
+        Raises a `ValueError` if the planner type is invalid.
```

Comment:
NBD but I do think `:raises` is a supported tag in Sphinx,

```suggestion
    :raises ValueError: if the planner type is invalid.
```